gradle/libs.versions.toml

```diff
@@ -25,7 +25,6 @@ flyway-core = { module = "org.flywaydb:flyway-core" }
 flyway-postgresql = { module = "org.flywaydb:flyway-database-postgresql" }
 google-maps-places = { module = "com.google.maps:google-maps-places", version.ref = "google-maps-places" }
 google-maps-services = { module = "com.google.maps:google-maps-services", version.ref = "google-maps-services" }
-jackson-datatype-jsr310 = { module = "com.fasterxml.jackson.datatype:jackson-datatype-jsr310" }
```

Comment:
This is already included in Jackson 3 by default

https://github.com/FasterXML/jackson-modules-java8?tab=readme-ov-file#jackson-3x

<img width="962" height="276" alt="Image" src="https://github.com/user-attachments/assets/04a8f424-5582-496b-b885-72362c1e140f" />